user_technical/how_to_guides/import-data.md

```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
 ```
 
 Next time you run Memgraph, the dataset will be loaded.
+
+### Importing Cypher Queries {#import-cypher}
+
+When Memgraph is running, cypher queries are imported by running `mg_client` in
+non-interactive mode. The user can import queries saved in e.g. `queries.txt`
+by issuing the following shell command:
+
+`mg_client < queries.txt`
```

Comment:
(view raw comment)

```bash
mg_client < queries.txt
```

This is not noticeable on github but its important for syntax highlighting on the web.